src/api/sessions.tsx

```diff
@@ -81,6 +81,7 @@ export const useDeleteSession = (): UseMutationResult<
     mutationFn: (session: SessionResponse) => deleteSession(session),
     onSuccess: async () => {
       await queryClient.invalidateQueries({ queryKey: ['sessionList'] });
+      queryClient.removeQueries({ queryKey: ['session'] });
```

Comment:
Should this not be more specific and only remove the query matching the session id of the one that was just deleted? Looking at the typing, `onSuccess` can get access to the `SessionResponse` object